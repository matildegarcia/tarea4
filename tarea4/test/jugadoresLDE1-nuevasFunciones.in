crearListaJugadores

# creo 6 jugadores y los agrego a la lista

# 1
crearPartida

crearJugada
5 3 1 2 3
agregarEnPartida

crearJugada
4 1 2 3 4
agregarEnPartida

crearJugador 10 23 Uruguay Pedro
crearFecha 12/3/2023
insertarJugadorListaJugadores



# 2
crearPartida

crearJugada
1 1 3 4 2
agregarEnPartida

crearJugada
3 1 2 1 2
agregarEnPartida

crearJugador 5 12 Uruguay Juana 
crearFecha 1/4/2019
insertarJugadorListaJugadores



# 3
crearPartida

crearJugada
6 2 2 1 4
agregarEnPartida

crearJugador 8 20 Uruguay Jose
crearFecha 25/1/2005
insertarJugadorListaJugadores


# 4
crearPartida

crearJugada
7 4 2 3 1
agregarEnPartida

crearJugada
9 1 3 2 1
agregarEnPartida

crearJugada
5 4 1 3 2
agregarEnPartida

crearJugador 13 64 Uruguay Antonio
crearFecha 17/8/2002
insertarJugadorListaJugadores


# 5
crearPartida

crearJugada
1 3 1 2 4
agregarEnPartida

crearJugador 20 33 Argentina Mario
crearFecha 11/2/1956
insertarJugadorListaJugadores


# 6
crearPartida

crearJugada
24 1 3 2 1
agregarEnPartida

crearJugada
31 1 4 2 2
agregarEnPartida

crearJugada
21 3 1 3 2
agregarEnPartida

crearJugada
4 3 2 1 1
agregarEnPartida

crearJugador 1 74 Chile Maria
crearFecha 7/4/1983
insertarJugadorListaJugadores

# imprimo la cantidad y la lista

imprimirCantidadListaJugadores
imprimirMayorAMenorTJugadoresLDE

# verifico que esten 4 jugadores distintos y los obtengo 

estaJugadorConNombreEnTJugadoresLDE Maria
obtenerJugadorConNombreTJugadoresLDE Maria

estaJugadorConNombreEnTJugadoresLDE Juana
obtenerJugadorConNombreTJugadoresLDE Juana

estaJugadorConNombreEnTJugadoresLDE Pedro
obtenerJugadorConNombreTJugadoresLDE Pedro

estaJugadorConNombreEnTJugadoresLDE Jose
obtenerJugadorConNombreTJugadoresLDE Jose

# ahora elimino a esos jugadores y verifico que no estan mas

eliminarJugadorConNombreTJugadoresLDE Maria
estaJugadorConNombreEnTJugadoresLDE Maria

eliminarJugadorConNombreTJugadoresLDE Juana
estaJugadorConNombreEnTJugadoresLDE Juana

eliminarJugadorConNombreTJugadoresLDE Pedro
estaJugadorConNombreEnTJugadoresLDE Pedro

eliminarJugadorConNombreTJugadoresLDE Jose
estaJugadorConNombreEnTJugadoresLDE Jose

# imprimo la cantidad y la lista

imprimirCantidadListaJugadores
imprimirMayorAMenorTJugadoresLDE

# libero la lista
liberarListaJugadores

Fin