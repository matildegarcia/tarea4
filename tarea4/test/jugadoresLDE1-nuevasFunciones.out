1>2>#  creo 6 jugadores y los agrego a la lista.
3>#  1.
4>5>6>7>8>9>10>11>12>#  2.
13>14>15>16>17>18>19>20>21>#  3.
22>23>24>25>26>27>28>#  4.
29>30>31>32>33>34>35>36>37>38>39>#  5.
40>41>42>43>44>45>46>#  6.
47>48>49>50>51>52>53>54>55>56>57>58>59>#  imprimo la cantidad y la lista.
60>La cantidad de jugadores en la lista es: 6.
61>Jugador 10: Pedro, Uruguay, 23 años
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugada: 5 
Movimientos: Arriba Derecha Izquierda Arriba 
12/3/2023
Jugador 5: Juana, Uruguay, 12 años
Jugada: 1 
Movimientos: Derecha Arriba Abajo Izquierda 
Jugada: 3 
Movimientos: Derecha Izquierda Derecha Izquierda 
1/4/2019
Jugador 8: Jose, Uruguay, 20 años
Jugada: 6 
Movimientos: Izquierda Izquierda Derecha Abajo 
25/1/2005
Jugador 13: Antonio, Uruguay, 64 años
Jugada: 5 
Movimientos: Abajo Derecha Arriba Izquierda 
Jugada: 7 
Movimientos: Abajo Izquierda Arriba Derecha 
Jugada: 9 
Movimientos: Derecha Arriba Izquierda Derecha 
17/8/2002
Jugador 1: Maria, Chile, 74 años
Jugada: 4 
Movimientos: Arriba Izquierda Derecha Derecha 
Jugada: 21 
Movimientos: Arriba Derecha Arriba Izquierda 
Jugada: 24 
Movimientos: Derecha Arriba Izquierda Derecha 
Jugada: 31 
Movimientos: Derecha Abajo Izquierda Izquierda 
7/4/1983
Jugador 20: Mario, Argentina, 33 años
Jugada: 1 
Movimientos: Arriba Derecha Izquierda Abajo 
11/2/1956
62>#  verifico que esten 4 jugadores distintos y los obtengo .
63>El jugador con nombre Maria está en la lista.
64>Jugador 1: Maria, Chile, 74 años
Jugada: 4 
Movimientos: Arriba Izquierda Derecha Derecha 
Jugada: 21 
Movimientos: Arriba Derecha Arriba Izquierda 
Jugada: 24 
Movimientos: Derecha Arriba Izquierda Derecha 
Jugada: 31 
Movimientos: Derecha Abajo Izquierda Izquierda 
65>El jugador con nombre Juana está en la lista.
66>Jugador 5: Juana, Uruguay, 12 años
Jugada: 1 
Movimientos: Derecha Arriba Abajo Izquierda 
Jugada: 3 
Movimientos: Derecha Izquierda Derecha Izquierda 
67>El jugador con nombre Pedro está en la lista.
68>Jugador 10: Pedro, Uruguay, 23 años
Jugada: 4 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugada: 5 
Movimientos: Arriba Derecha Izquierda Arriba 
69>El jugador con nombre Jose está en la lista.
70>Jugador 8: Jose, Uruguay, 20 años
Jugada: 6 
Movimientos: Izquierda Izquierda Derecha Abajo 
71>#  ahora elimino a esos jugadores y verifico que no estan mas.
72>73>El jugador con nombre Maria NO está en la lista.
74>75>El jugador con nombre Juana NO está en la lista.
76>77>El jugador con nombre Pedro NO está en la lista.
78>79>El jugador con nombre Jose NO está en la lista.
80>#  imprimo la cantidad y la lista.
81>La cantidad de jugadores en la lista es: 2.
82>Jugador 13: Antonio, Uruguay, 64 años
Jugada: 5 
Movimientos: Abajo Derecha Arriba Izquierda 
Jugada: 7 
Movimientos: Abajo Izquierda Arriba Derecha 
Jugada: 9 
Movimientos: Derecha Arriba Izquierda Derecha 
17/8/2002
Jugador 20: Mario, Argentina, 33 años
Jugada: 1 
Movimientos: Arriba Derecha Izquierda Abajo 
11/2/1956
83>#  libero la lista.
84>85>Fin.
